libs/testing/src/jobs.service.ts

```diff
@@ -0,0 +1,55 @@
+import { Queue } from 'bullmq';
+import { JobRepository, JobStatusEnum } from '@novu/dal';
+import { StepTypeEnum } from '@novu/shared';
+
+import { QueueService } from './queue.service';
+
+export class JobsService {
```

Comment:
new class for the testing purpose, two functions were moved from the UserSession class: `awaitParsingEvents, awaitRunningJobs`

Reply:
Great!